### ISSUE TITLE
thetvdb_favorites: 403 Client Error

### ISSUE THREAD

**Issue by elgfaen** [Flexget/Flexget#1137]
I updated to Flexget 2.0.10 today and discovered that there has been a change to thetvdb api and thetvdb_favorites plugin. I've now altered the config and provided the username and account identifier.

The plugin is stil unable to access my favorites-list, and throws a 403 error:

Error retrieving favorites from thetvdb: 403 Client Error: Forbidden for url: https://api.thetvdb.com/user/favorites

Have anyone else encountered this?

**Comment by stevezau** [comment]
Strange, that should work, suggest fail auth... Have you tried since?

**Comment by elgfaen** [comment]
I've done some more testing today. I've created a new account at thetvdb.com and added some shows as favorites. I've then added the new username and account identifier to the config under the 'tv-2' task. This is what I get when executing flexget:

'tv-1' task:
```
2016-05-06 22:31 ERROR    thetvdb_favorites tv-1        Error retrieving favorites from thetvdb: 403 Client Error: Forbidden for url: https://api.thetvdb.com/user/favorites
2016-05-06 22:31 WARNING  configure_series tv-1        Input thetvdb_favorites did not return anything
2016-05-06 22:31 INFO     configure_series tv-1        Did not get any series to generate series configuration
```

'tv-2' task:
```
2016-05-06 22:31 ERROR    thetvdb_favorites tv-2        Error retrieving favorites from thetvdb: 403 Client Error: Forbidden for url: https://api.thetvdb.com/login
2016-05-06 22:31 WARNING  configure_series tv-2        Input thetvdb_favorites did not return anything
2016-05-06 22:31 INFO     configure_series tv-2        Did not get any series to generate series configuration
```

So it would seem that the 'tv-1' task makes it though authentication (https://api.thetvdb.com/login) but then gets served a 403 while trying to get the favorites. The second task falt out fails authentication, even tho I've copied the username and account identifier from thetvdb.com. Correct me if I'm wrong, but from what I understand it is no longer required that you supply thetvdb_favorites with your own API-key. However, if I use my own api-key with curl, I'm able to retrieve the favorites-list:

 ```
curl -XPOST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '{"apikey":"XXXXX","username":"XXXX","userkey":"XXXXX"}' 'https://api.thetvdb.com/login'
curl -X GET --header 'Accept: application/json' --header 'Authorization: Bearer loooongkeyasdhjahsdjashdk' 'https://api.thetvdb.com/user/favorites'
{
  "data": {
    "favorites": [
      "286193",
      "277165",
      "270633",
      "247897",
      "274431",
      "303210",
      "77398",
      "289590",
      "295829",
      "290853",
      "298085"
    ]
  }
```

Do thetvdb_favorites have a static api-key which is used when autenticating? If so, how would this fly with api.thetvdb.com when the api-key don't match with username and account identifier?

**Comment by crawln45** [comment]
Odd. I just did this with mine. On 2.0.11.dev.

**Comment by stevezau** [comment]
hmm, are you using a different user between tasks? Can u run it in debug and paste the output

**Comment by elgfaen** [comment]
I'm running two tasks and both have their own tvdb-account. Here is the flexget -L debug:

Task 1:
```
2016-05-08 00:07 DEBUG    task          tv-panne        executing tv-panne
2016-05-08 00:07 INFO     ipc.rpyc                      server started on [127.0.0.1]:45272
2016-05-08 00:07 DEBUG    template      tv-panne        Merging template transmission-generell into task tv-panne
2016-05-08 00:07 DEBUG    template      tv-panne        Merging template tv-panne into task tv-panne
2016-05-08 00:07 DEBUG    template      tv-panne        Merging template rensk-transmission into task tv-panne
2016-05-08 00:07 DEBUG    input_cache   tv-panne        cache name: thetvdb_favorites_474102ef4d0d84e7f01c824b43fa1885 (has: )
2016-05-08 00:07 DEBUG    util.simple_persistence tv-panne        setting key auth_tokens value {u'default': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTQ4ODAsImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY4NDgwfQ.F8tFMQU1hlruKeam35qPzWOBBRJoIi88zHAcW5LUMUdvALtA15nf-5Ai7gED75-DVmOvD8ddgl5nxoSY5tRMXdbjoPPhNAOirQWnLjq2Tz7C4t8ZGfzwSjh51dt2VBOsguVnOvhC6cBbSUBSLL6z1CYdXezFh0oEfus-BISHO4LVgFO4G0Ov9WBo3xD5-VQa5EJjusp05YibjXLd4nAGz4IkMOxOFgzF7yoKwER1siRFDByqr2UYQzyzTUd2zeW5aPdeONMSfO_ZPX1ikJE308e-F-D-2kuj8uWQuEtbxus9YvIQ8C4LtSNLIfdMDnYfEV8hi3ef1fh3Zwsws4miiw', u'Kastela': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTEyNzksImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY0ODc5LCJ1c2VyaWQiOjQzNDMzOSwidXNlcm5hbWUiOiJLYXN0ZWxhIn0.ayRelEn4F4jnBscPH0xSfW4I_erBLHckLgDCSLDy0y_H_kv3VAcwUTGFaHq1YXwzSGeTvUv9pvL7ghKqDHl3Gvzt4KaURr9NzDGZoQ36fJ54IBnyuh0Zx7J4cOaHOqdQPSDYpAblPpBuFG0Ya6dmNB3mqn_wrCPICkE0gP6g7izWiQm2GPggBNlwJHti9fKTGEoqySMNHuqUGhN33YWSATGobEwjLFS-eCkVoUdwiY6BdCMnIllhehp1Rzy8T0EtGha9laHb6ovwkCcluzpz0DJ-LYIGSvMndmYtQzJJrxsanTMpvPUIXcyqk6LPNNy2CagCd1qlrlTIt637_p833A', u'elgfaen': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTEyODMsImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY0ODgzLCJ1c2VyaWQiOjQyMzA3NywidXNlcm5hbWUiOiJlbGdmYWVuIn0.Htnoc_zHO-mwDeYY5pOEf-jH0YGhQKRBSwe85Rdv6HefncqgBFIpTfShA7tidfapOb8QQJPtjzEcoBv7_-H6aVmzbX0Y6bp10pH4kqigC_McbLAB40l_TOLAgZidOCQ-WMMI9qPWD000GV3CeZ48I3cSZ1y7xaYhprVOy70NlpI_v4Lv-Cg5M_bUNmG0W-VqOQfnDLWiQdmg8PcVHihLtJvpbPzKMmnBAfzY-3Ndn-eC7FbZMIc5qi3bjI9VDf2ehZJeXj477klCGGZ9kOldK5UmQ3jNNMAyFXv3Z-VcRxSILfIFYutBZbJF3ZOJH1Kk6or5Iqk4uRpdyiktKqepQw'}
2016-05-08 00:07 DEBUG    utils.requests tv-panne        Fetching https://api.thetvdb.com/user/favorites
2016-05-08 00:07 ERROR    thetvdb_favorites tv-panne        Error retrieving favorites from thetvdb: 403 Client Error: Forbidden for url: https://api.thetvdb.com/user/favorites
2016-05-08 00:07 DEBUG    input_cache   tv-panne        storing to cache thetvdb_favorites_474102ef4d0d84e7f01c824b43fa1885 0 entries
2016-05-08 00:07 WARNING  configure_series tv-panne        Input thetvdb_favorites did not return anything
2016-05-08 00:07 INFO     configure_series tv-panne        Did not get any series to generate series configuration
2016-05-08 00:07 DEBUG    input_cache   tv-panne        cache name: rss_df8c2fdcdea882c2770d68adc80e9056 (has: thetvdb_favorites_474102ef4d0d84e7f01c824b43fa1885)
2016-05-08 00:07 DEBUG    rss           tv-panne        Requesting task `tv-panne` url `https://rarbg.com/rssdd.php?category=1;18;41`
2016-05-08 00:07 DEBUG    utils.requests tv-panne        Fetching https://rarbg.com/rssdd.php?category=1;18;41
2016-05-08 00:07 DEBUG    rss           tv-panne        encoding utf-8
2016-05-08 00:07 DEBUG    rss           tv-panne        Saving location in rss feed.
2016-05-08 00:07 DEBUG    util.simple_persistence tv-panne        setting key -3460365637297962714_last_entry value u'Britains.Got.Talent.S10E05.720p.HDTV.x264-FTP[rartv]https://rarbg.com/download.php?id=78e1rhb&f=Britains.Got.Talent.S10E05.720p.HDTV.x264-FTP[rartv].torrent&p=r'
2016-05-08 00:07 DEBUG    input_cache   tv-panne        storing to cache rss_df8c2fdcdea882c2770d68adc80e9056 25 entries
2016-05-08 00:07 DEBUG    backlog       tv-panne        0 entries purged from backlog
2016-05-08 00:07 VERBOSE  details       tv-panne        Produced 25 entries.
2016-05-08 00:07 DEBUG    exists_series tv-panne        Scanning not needed
2016-05-08 00:07 DEBUG    urlrewriter   tv-panne        Checking 0 entries
2016-05-08 00:07 VERBOSE  details       tv-panne        Summary - Accepted: 0 (Rejected: 0 Undecided: 25 Failed: 0)
2016-05-08 00:07 VERBOSE  email         tv-panne        No content generated from template, not sending email.
2016-05-08 00:07 VERBOSE  transmission  tv-panne        Torrent "The.Thirteenth.Floor.1999.1080p.BluRay.AVC.TrueHD.5.1-FGT": status: "downloading" - ratio: 0.6501 -  date added: 2016-05-07 23:00:56 - date done: 1970-01-01 01:00:00
```

Task 2:
```
2016-05-08 00:07 DEBUG    task          tv-lasse        executing tv-lasse
2016-05-08 00:07 DEBUG    template      tv-lasse        Merging template transmission-generell into task tv-lasse
2016-05-08 00:07 DEBUG    template      tv-lasse        Merging template tv-lasse into task tv-lasse
2016-05-08 00:07 DEBUG    template      tv-lasse        Merging template rensk-transmission into task tv-lasse
2016-05-08 00:07 DEBUG    input_cache   tv-lasse        cache name: thetvdb_favorites_e3b2a495745b0ca2ab7926078d046c4e (has: thetvdb_favorites_474102ef4d0d84e7f01c824b43fa1885, rss_f04c26c3892ec8300888917dbe3ea151, rss_df8c2fdcdea882c2770d68adc80e9056)
2016-05-08 00:07 DEBUG    util.simple_persistence tv-lasse        setting key auth_tokens value {u'default': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTQ4ODAsImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY4NDgwfQ.F8tFMQU1hlruKeam35qPzWOBBRJoIi88zHAcW5LUMUdvALtA15nf-5Ai7gED75-DVmOvD8ddgl5nxoSY5tRMXdbjoPPhNAOirQWnLjq2Tz7C4t8ZGfzwSjh51dt2VBOsguVnOvhC6cBbSUBSLL6z1CYdXezFh0oEfus-BISHO4LVgFO4G0Ov9WBo3xD5-VQa5EJjusp05YibjXLd4nAGz4IkMOxOFgzF7yoKwER1siRFDByqr2UYQzyzTUd2zeW5aPdeONMSfO_ZPX1ikJE308e-F-D-2kuj8uWQuEtbxus9YvIQ8C4LtSNLIfdMDnYfEV8hi3ef1fh3Zwsws4miiw', u'Kastela': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTEyNzksImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY0ODc5LCJ1c2VyaWQiOjQzNDMzOSwidXNlcm5hbWUiOiJLYXN0ZWxhIn0.ayRelEn4F4jnBscPH0xSfW4I_erBLHckLgDCSLDy0y_H_kv3VAcwUTGFaHq1YXwzSGeTvUv9pvL7ghKqDHl3Gvzt4KaURr9NzDGZoQ36fJ54IBnyuh0Zx7J4cOaHOqdQPSDYpAblPpBuFG0Ya6dmNB3mqn_wrCPICkE0gP6g7izWiQm2GPggBNlwJHti9fKTGEoqySMNHuqUGhN33YWSATGobEwjLFS-eCkVoUdwiY6BdCMnIllhehp1Rzy8T0EtGha9laHb6ovwkCcluzpz0DJ-LYIGSvMndmYtQzJJrxsanTMpvPUIXcyqk6LPNNy2CagCd1qlrlTIt637_p833A', u'elgfaen': u'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NjA4NTEyODMsImlkIjoiRmxleEdldCIsIm9yaWdfaWF0IjoxNDYwNzY0ODgzLCJ1c2VyaWQiOjQyMzA3NywidXNlcm5hbWUiOiJlbGdmYWVuIn0.Htnoc_zHO-mwDeYY5pOEf-jH0YGhQKRBSwe85Rdv6HefncqgBFIpTfShA7tidfapOb8QQJPtjzEcoBv7_-H6aVmzbX0Y6bp10pH4kqigC_McbLAB40l_TOLAgZidOCQ-WMMI9qPWD000GV3CeZ48I3cSZ1y7xaYhprVOy70NlpI_v4Lv-Cg5M_bUNmG0W-VqOQfnDLWiQdmg8PcVHihLtJvpbPzKMmnBAfzY-3Ndn-eC7FbZMIc5qi3bjI9VDf2ehZJeXj477klCGGZ9kOldK5UmQ3jNNMAyFXv3Z-VcRxSILfIFYutBZbJF3ZOJH1Kk6or5Iqk4uRpdyiktKqepQw'}
2016-05-08 00:07 DEBUG    utils.requests tv-lasse        Fetching https://api.thetvdb.com/user/favorites
2016-05-08 00:07 ERROR    thetvdb_favorites tv-lasse        Error retrieving favorites from thetvdb: 403 Client Error: Forbidden for url: https://api.thetvdb.com/user/favorites
2016-05-08 00:07 DEBUG    input_cache   tv-lasse        storing to cache thetvdb_favorites_e3b2a495745b0ca2ab7926078d046c4e 0 entries
2016-05-08 00:07 WARNING  configure_series tv-lasse        Input thetvdb_favorites did not return anything
2016-05-08 00:07 INFO     configure_series tv-lasse        Did not get any series to generate series configuration
2016-05-08 00:07 DEBUG    input_cache   tv-lasse        cache name: rss_df8c2fdcdea882c2770d68adc80e9056 (has: thetvdb_favorites_474102ef4d0d84e7f01c824b43fa1885, thetvdb_favorites_e3b2a495745b0ca2ab7926078d046c4e, rss_f04c26c3892ec8300888917dbe3ea151, rss_df8c2fdcdea882c2770d68adc80e9056)
2016-05-08 00:07 VERBOSE  input_cache   tv-lasse        Restored 25 entries from cache
2016-05-08 00:07 DEBUG    backlog       tv-lasse        0 entries purged from backlog
2016-05-08 00:07 VERBOSE  details       tv-lasse        Produced 25 entries.
2016-05-08 00:07 DEBUG    exists_series tv-lasse        Scanning not needed
2016-05-08 00:07 DEBUG    urlrewriter   tv-lasse        Checking 0 entries
2016-05-08 00:07 VERBOSE  details       tv-lasse        Summary - Accepted: 0 (Rejected: 0 Undecided: 25 Failed: 0)
2016-05-08 00:07 VERBOSE  email         tv-lasse        No content generated from template, not sending email.
2016-05-08 00:07 VERBOSE  transmission  tv-lasse        Torrent "The.Thirteenth.Floor.1999.1080p.BluRay.AVC.TrueHD.5.1-FGT": status: "downloading" - ratio: 0.6504 -  date added: 2016-05-07 23:00:56 - date done: 1970-01-01 01:00:00
2016-05-08 00:07 DEBUG    util.simple_persistence tv-lasse        Flushing simple persistence for task tv-lasse to db.
```

Hm, so it now seems I'm able to /login and get a token, but that token is not valid for some reason. Also, the same token is used for both accounts. Shouldn't each account get their own unique token? Is there some form of caching going on here with the tokens?

**Comment by elgfaen** [comment]
Ok, I found out whats going on here. Did a call to the api againg with curl. It seems that because I run the network activity through a VPN the new tvdb-api serves me a captcha to solve. Tada:

```
<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
<title>Attention Required! | CloudFlare</title>
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection" />
<!--[if lt IE 9]><link rel="stylesheet" id='cf_styles-ie-css' href="/cdn-cgi/styles/cf.errors.ie.css" type="text/css" media="screen,projection" /><![endif]-->
<style type="text/css">body{margin:0;padding:0}</style>
<!--[if lte IE 9]><script type="text/javascript" src="/cdn-cgi/scripts/jquery.min.js"></script><![endif]-->
<!--[if gte IE 10]><!--><script type="text/javascript" src="/cdn-cgi/scripts/zepto.min.js"></script><!--<![endif]-->
<script type="text/javascript" src="/cdn-cgi/scripts/cf.common.js"></script>


</head>
<body>
  <div id="cf-wrapper">
    <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
    <div id="cf-error-details" class="cf-error-details-wrapper">
      <div class="cf-wrapper cf-header cf-error-overview">
        <h1 data-translate="challenge_headline">One more step</h1>
        <h2 class="cf-subheadline"><span data-translate="complete_sec_check">Please complete the security check to access</span> api.thetvdb.com</h2>
      </div><!-- /.header -->

      <div class="cf-section cf-highlight cf-captcha-container">
        <div class="cf-wrapper">
          <div class="cf-columns two">
            <div class="cf-column">
              <div class="cf-highlight-inverse cf-form-stacked">
                <form class="challenge-form" id="challenge-form" action="/cdn-cgi/l/chk_captcha" method="get">
  <script type="text/javascript" src="/cdn-cgi/scripts/cf.challenge.js" data-type="normal"  data-ray="29f8275a43cc0749" async data-sitekey="6LfOYgoTAAAAAInWDVTLSc8Yibqp-c9DaLimzNGM" data-stoken="w544lymazwGjwIrzS5FkbUqBZBaEVk7TFNzhkuZ1x1UmsmEWJsjHpcR5K_9dM8f9P1jiXu-4JN444NLQH1KDkZX0VZ0dRkPTPDmR033Eg14"></script>
  <div class="g-recaptcha"></div>
  <noscript id="cf-captcha-bookmark" class="cf-captcha-info">
    <div><div style="width: 302px">
      <div>
        <iframe src="https://www.google.com/recaptcha/api/fallback?k=6LfOYgoTAAAAAInWDVTLSc8Yibqp-c9DaLimzNGM&stoken=w544lymazwGjwIrzS5FkbUqBZBaEVk7TFNzhkuZ1x1UmsmEWJsjHpcR5K_9dM8f9P1jiXu-4JN444NLQH1KDkZX0VZ0dRkPTPDmR033Eg14" frameborder="0" scrolling="no" style="width: 302px; height:422px; border-style: none;"></iframe>
      </div>
      <div style="width: 300px; border-style: none; bottom: 12px; left: 25px; margin: 0px; padding: 0px; right: 25px; background: #f9f9f9; border: 1px solid #c1c1c1; border-radius: 3px;">
        <textarea id="g-recaptcha-response" name="g-recaptcha-response" class="g-recaptcha-response" style="width: 250px; height: 40px; border: 1px solid #c1c1c1; margin: 10px 25px; padding: 0px; resize: none;"></textarea>
        <input type="submit" value="Submit"></input>
      </div>
    </div></div>
  </noscript>
</form>

              </div>
            </div>

            <div class="cf-column">
              <div class="cf-screenshot-container">

                <span class="cf-no-screenshot"></span>

              </div>
            </div>
          </div><!-- /.columns -->
        </div>
      </div><!-- /.captcha-container -->

      <div class="cf-section cf-wrapper">
        <div class="cf-columns two">
          <div class="cf-column">
            <h2 data-translate="why_captcha_headline">Why do I have to complete a CAPTCHA?</h2>

            <p data-translate="why_captcha_detail">Completing the CAPTCHA proves you are a human and gives you temporary access to the web property.</p>
          </div>

          <div class="cf-column">
            <h2 data-translate="resolve_captcha_headline">What can I do to prevent this in the future?</h2>

            <p data-translate="resolve_captcha_antivirus">If you are on a personal connection, like at home, you can run an anti-virus scan on your device to make sure it is not infected with malware.</p>

            <p data-translate="resolve_captcha_network">If you are at an office or shared network, you can ask the network administrator to run a scan across the network looking for misconfigured or infected devices.</p>
          </div>
        </div>
      </div><!-- /.section -->

      <div class="cf-error-footer cf-wrapper">
  <p>
    <span class="cf-footer-item">CloudFlare Ray ID: <strong>29f8275a43cc0749</strong></span>
    <span class="cf-footer-separator">&bull;</span>
    <span class="cf-footer-item"><span data-translate="your_ip">Your IP</span>: 46.166.186.238</span>
    <span class="cf-footer-separator">&bull;</span>
    <span class="cf-footer-item"><span data-translate="performance_security_by">Performance &amp; security by</span> <a data-orig-proto="https" data-orig-ref="www.cloudflare.com/5xx-error-landing?utm_source=error_footer" id="brand_link" target="_blank">CloudFlare</a></span>

  </p>
</div><!-- /.error-footer -->


    </div><!-- /#cf-error-details -->
  </div><!-- /#cf-wrapper -->

  <script type="text/javascript">
  window._cf_translation = {};


</script>

</body>
</html>

```

API with a captcha, thats cool. I guess I'll close this case and complain to the tvdb-guys instead :P

**Comment by liiight** [comment]
Nice, thanks for reporting. I wonder if it's something worth catching in
the code and handle better though. What do you think @stevezau?

**Comment by elgfaen** [comment]
No problem. And thank you all for being helpfull :)

**Comment by rpatterson** [comment]
I've been getting clobbered with this error.  The root issue is that VPN IPs are pretty much guaranteed to be designated as "suspicious" by CloudFlare and thus be presented with a captcha challenge.  That challenge uses cookies to prevent subsequent requests from getting the captcha challenge.  So the only real solution is to include those cookies in the requests that Flexget sends.

Then I discovered that Flexget has support for including cookies in requests via [the `cookies` plugin](http://flexget.com/Plugins/cookies) and I added that plugin to each of my Flexget tasks that sends any requests.   So now, when I start seeing `403 Forbidden` errors coming from any of the requests that Flexget sends, I fire up Firefox with a designated "clean" profile used just for Flexget, answer the captcha, and copy the Firefox profile's `cookies.sqlite` file to the path designated in the `cookies` plugin config YAML.

Since I'm running Flexget on a headless server, I needed to find a way to run that Firefox profile as if it were running on that server.  First, I tried using SSH X forwarding, but the triple latency of SSH+Server+VPN made Firefox so unusable I couldn't even complete the captchas.  I ended up using the SSH SOCKS proxy support (`ssh -D 1080 ...`) and running the Firefox profile under `tsocks`  which is usable enough to complete the captchas.

Hope this is useful to others.

**Comment by ricka** [comment]
@rpatterson I'm having the same problem with CloudFlare and captcha.  I've tried setting up the cookies plugin, but I'm continually getting 403 errors.  I was wondering if you wouldn't mind sharing your flexget config?

Additionally, I've done the same by doing `ssh -D 9999 ...` and using the FoxyProxy addon.  I've looked in the cookies.sqlite file and the correct cookies seem to be set, but still no dice.  Any other thoughts on what to troubleshoot?
